app/src/main/java/com/vultisig/wallet/ui/models/send/AmountManager.kt

```
@@ -0,0 +1,168 @@
+
+    private suspend fun convertValue(
+        value: String,
+        token: Coin,
+        transform: (value: BigDecimal, price: BigDecimal, token: Coin) -> BigDecimal,
+    ): String? {
+        val decimalValue = value.toBigDecimalOrNull() ?: return ""
+
+        val price =
+            try {
+                tokenPriceRepository.getPrice(token, appCurrency.value).first()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)